Source/CombatExtended/CombatExtended/CE_Utility.cs

```diff
@@ -112,6 +113,19 @@ public static Texture2D BlitCrop(this Texture2D texture, Rect blitRect)
         #region Misc
         public static List<ThingDef> allWeaponDefs = new List<ThingDef>();
 
+        public static readonly FieldInfo cachedLabelCapInfo = typeof(Def).GetField("cachedLabelCap", BindingFlags.NonPublic | BindingFlags.Instance);
+
+        public static void UpdateLabel(this Def def, string label)
+        {
+            if (label.NullOrEmpty())
+            {
+                Log.Error("ERROR: GunMod.UpdateLabel(this Def def, string label) was called with label.NullOrEmpty() for def \"" + def.defName + "\"");
```

Comment:
What's GunMod referring to?

Reply:
Oh right, I took this stuff from Rimfire's "GunMod" stuff

Reply:
Removed error logging (unnecessary in CE) in the latest commit - that's all that was done in the commit too.